irohad/ametsuchi/impl/rocksdb_command_executor.cpp

```diff
@@ -80,12 +82,14 @@ CommandResult RocksDbCommandExecutor::execute(
                               .assumeValue());
           }
 
-          return (*this)(command,
-                         creator_account_id,
-                         tx_hash,
-                         cmd_index,
-                         do_validation,
-                         creator_permissions);
+          auto result = (*this)(command,
+                                creator_account_id,
+                                tx_hash,
+                                cmd_index,
+                                do_validation,
+                                creator_permissions);
+          common.commit();
```

Comment:
This does not look like the right place to commit because we need to rollback commands from a transaction if some command fails.